src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -268,25 +268,31 @@
 (defn message
   [{:keys [pinned-by mentioned content-type last-in-group? deleted? deleted-for-me?]
     :as   message-data} {:keys [in-pinned-view?] :as context} keyboard-shown?]
-  [rn/view
-   {:style               (style/message-container in-pinned-view? pinned-by mentioned last-in-group?)
-    :accessibility-label :chat-item}
-   (cond
-     (system-message? content-type)
-     [system-message-content message-data]
+  (let [system-message? (get-system-message? content-type)]
```

Comment:
`(check-fi-system-message content-type)` idk might be better.

Reply:
done